oscarlab/graphene
# [meson] Compile Pal/Skeleton

## Description of the changes 

This is another PR in the meson series. I want to merge this before SGX (#2411), because then it will be easier to untangle some Makefile mess around `/Pal/**Linux-SGX/tools/common` vs `/common` vs `LibOS/shim/test/*`.

## How to test this PR? 

```sh
make -C Pal/src PAL_HOST=Skeleton
meson build -Dskeleton=enabled
```
Tests won't run, because everything fails by desing, though `libpal.so` is installed (you can link against it, and if skeleton is enabled, LibOS is linked against it, because should compiles fastest).


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/oscarlab/graphene/2418)
